live/tests/unit/models/assessment-progress-test.js

```diff
@@ -0,0 +1,130 @@
+import { expect } from 'chai';
+import { describe, context, it, beforeEach } from 'mocha';
+import AssessmentProgression from 'pix-live/models/assessment-progression';
+
+describe('Unit | Model | assessment progress', function() {
+
+  describe('@_currentStep', function() {
+
+    context('when assessment type is not "SMART_PLACEMENT"', function() {
+
+      it('should return the number of answers linked to an assessment', () => {
```

Comment:
should return the number of current challenge ? 
Vu qu'on retourne 9 pour 8 answers ?